Makefile

```diff
@@ -0,0 +1,3 @@
+lint:
+	@./node_modules/.bin/eslint .
```

Comment:
How about combine this to webpack? since we already using that.
Seems like only koa use Makefile, maybe it's using Makefile for some other purpose than just lint or test.
I start thinking this is just a preference problem.